app/controllers/coronavirus_form/session_controller.rb

```diff
@@ -3,6 +3,7 @@
 class CoronavirusForm::SessionController < ApplicationController
   def delete
     reset_session
-    redirect_to "/"
+    redirect_to I18n.t("leave_this_website.link_redirect_to") if params[:ext_r] == "true"
+    redirect_to "/" if params[:ext_r] != "true"
```

Comment:
```suggestion
    if params[:ext_r] == "true"
        redirect_to I18n.t("leave_this_website.link_redirect_to")
    else
        redirect_to "/"
    end
```

Reply:
yeah cool, makes more sense. amended

Reply:
Pretty sure you didn't need the second if. I think this could have been written as:

```
redirect_to I18n.t("leave_this_website.link_redirect_to") if params[:ext_r] == "true"
redirect_to "/"
```